src/Microsoft.DotNet.Darc/src/DarcLib/Models/VirtualMonoRepo/ManifestRecord.cs

```diff
@@ -9,20 +9,39 @@ namespace Microsoft.DotNet.Darc.Models.VirtualMonoRepo;
 
 public abstract class ManifestRecord : IComparable<ManifestRecord>
```

Comment:
Please add a short XML comment explaining in what scenario is this class used (or the other two below)